index.md

```diff
@@ -58,46 +82,25 @@ Here's the video for 0.3:
 
 <br>
 
-## Installation and first steps
+## Send us your feedback!
 
-The pre-requisites for RTVS are: 
+We have three different ways you can send feedback to the team:
 
-* Visual Studio 2015 Community, Professional or Enterprise
-* Visual Studio 2015 Update 3
-* An R interpreter: CRAN-R or Microsoft R Open
+1. **Via Github**: This is the preferred way to send us feedback. You can use
+   [this link](https://github.com/Microsoft/RTVS/issues), or the built-in
+   shortcut in the RTools -> Feedback menu.
 
-Please check out the installation steps, *especially* if you are new to Visual
-Studio
-
-* [RTVS Installation](installation.html) 
-
-Once installed, create a project and code away!
-
-* [How to start a new project in RTVS](start-project.html)
-
-## Feedback, bugs, etc.
-
-Please file bugs and feature requests directly on:
-
-* [RTVS on Github](https://github.com/Microsoft/RTVS)
-
-To send us bug repros (which are highly appreciated) try the built in
-Smile/Frown feature:
+1. **Send a Smile / Frown**: This is a quick way to send feedback **and** attach
+RTVS log files to assist in the diagnosis of your issue. You can find this
+command under the R Tools -> Feedback menu. This command will collect logs,
+start your mail client and attach the log file. You have the opportunity to
+examine the contents of those files before you click Send. The logs are written
+into `%TEMP%/RTVSlogs.zip` in case you want to send it yourself.
```

Comment:
This feature is not available if user opted out of VS telemetry in Help | Feedback | Settings.